### PR TITLE
Convert "Orbit" references to "Astro-UI"

### PR DESCRIPTION
**Pull request by paolaperaza**
Resolves: https://github.com/astronomer/docs/issues/275

Adding a PR here that addresses all legacy references to "Orbit UI", which used to be the name of the Astronomer UI component and corresponding pods in Kubernetes. All instances of `orbit-ui` should be `astro-ui`.

### CODE REVIEW

File: enterprise/next/03_install/01_aws/01_install-aws-standard.md [hunk 1]
```diff
@@ -364,7 +364,7 @@ astronomer-kube-state                ClusterIP      172.20.123.56    <none>
 astronomer-kubed                     ClusterIP      172.20.4.200     <none>                                                                    443/TCP                                      24d
 astronomer-nginx                     LoadBalancer   172.20.54.142    ELB_ADDRESS.us-east-1.elb.amazonaws.com                                   80:31925/TCP,443:32461/TCP,10254:32424/TCP   24d
 astronomer-nginx-default-backend     ClusterIP      172.20.186.254   <none>                                                                    8080/TCP                                     24d
-astronomer-orbit                     ClusterIP      172.20.186.166   <none>                                                                    8080/TCP                                     24d
+astro-ui                     ClusterIP      172.20.186.166   <none>                                                                    8080/TCP                                     24d
```
Comment (paolaperaza) [inline enterprise/next/03_install/01_aws/01_install-aws-standard.md:367]: Should this be `astronomer-astro-ui`?

File: enterprise/v0.12/06_manage-astronomer/08_configure-platform-resources.md [hunk 2]
```diff
@@ -35,14 +35,14 @@ By default, Astronomer needs around 10 CPUs and 44Gi of memory:
...
+To change something like the resources allocated to `astro-ui`, add the following fields to your `config.yaml`:
 ```
 #####
-#Changing Orbit CPU
+#Changing Astronomer UI CPU
 ####
 
 astronomer:
-  orbit:
+  astro-ui:
```
Comment (schnie) [inline enterprise/v0.12/06_manage-astronomer/08_configure-platform-resources.md:45]:
```suggestion
  astroUI:
```

Comment (schnie) [reply enterprise/v0.12/06_manage-astronomer/08_configure-platform-resources.md:45]: Maps to https://github.com/astronomer/astronomer/blob/master/values.yaml#L134

File: enterprise/v0.13/06_manage-astronomer/08_configure-platform-resources.md [hunk 3]
```diff
@@ -35,14 +35,14 @@ By default, Astronomer needs around 10 CPUs and 44Gi of memory:
...
+To change something like the resources allocated to `astro-ui`, add the following fields to your `config.yaml`:
 ```
 #####
-#Changing Orbit CPU
+#Changing Astronomer UI CPU
 ####
 
 astronomer:
-  orbit:
+  astro-ui:
```
Comment (schnie) [inline enterprise/v0.13/06_manage-astronomer/08_configure-platform-resources.md:45]:
```suggestion
  astroUI:
```

File: enterprise/v0.15/06_manage-astronomer/08_configure-platform-resources.md [hunk 4]
```diff
@@ -35,14 +35,14 @@ By default, Astronomer needs around 10 CPUs and 44Gi of memory:
...
+To change something like the resources allocated to `astro-ui`, add the following fields to your `config.yaml`:
 ```
 #####
-#Changing Orbit CPU
+#Changing Astronomer UI CPU
 ####
 
 astronomer:
-  orbit:
+  astro-ui:
```
Comment (schnie) [inline enterprise/v0.15/06_manage-astronomer/08_configure-platform-resources.md:45]:
```suggestion
  astroUI:
```

File: enterprise/v0.16/06_manage-astronomer/09_configure-platform-resources.md [hunk 5]
```diff
@@ -35,15 +35,15 @@ By default, Astronomer needs around 10 CPUs and 44Gi of memory:
...
 
 ```
 #####
-#Changing Orbit CPU
+#Changing Astronomer UI CPU
 ####
 
 astronomer:
-  orbit:
+  astro-ui:
```
Comment (schnie) [inline enterprise/v0.16/06_manage-astronomer/09_configure-platform-resources.md:46]:
```suggestion
  astroUI:
```

File: enterprise/v0.23/06_manage-astronomer/09_configure-platform-resources.md [hunk 6]
```diff
@@ -35,15 +35,15 @@ By default, Astronomer needs around 10 CPUs and 44Gi of memory:
...
 
 ```
 #####
-#Changing Orbit CPU
+#Changing Astronomer UI CPU
 ####
 
 astronomer:
-  orbit:
+  astro-ui:
```
Comment (schnie) [inline enterprise/v0.23/06_manage-astronomer/09_configure-platform-resources.md:46]:
```suggestion
  astroUI:
```

File: enterprise/next/06_manage-astronomer/08_configure-platform-resources.md [hunk 7]
```diff
@@ -35,15 +35,15 @@ By default, Astronomer needs around 10 CPUs and 44Gi of memory:
...
 
 ```
 #####
-#Changing Orbit CPU
+#Changing Astronomer UI CPU
 ####
 
 astronomer:
-  orbit:
+  astro-ui:
```
Comment (paolaperaza) [inline enterprise/next/06_manage-astronomer/08_configure-platform-resources.md:46]:
```suggestion
  astroUI:
```

**Review by schnie — APPROVED** [review]

File: enterprise/next/03_install/01_aws/01_install-aws-standard.md (hunk 1 above)
Comment (paolaperaza) [reply enterprise/next/03_install/01_aws/01_install-aws-standard.md]: Confirmed that yes! Changed.

**Pull request merged**